src/components/MySpace/MySpace.tsx

```diff
@@ -100,109 +115,141 @@ const MySpace = ({ bookmarks }: { bookmarks: BookmarkRecords }) => {
 
           {data &&
             data.mySpace &&
-            data.mySpace.map((widget: Widget) => (
-              <Grid
-                key={`widget_${widget._id}`}
-                tabletLg={{ col: 6 }}
-                desktopLg={{ col: 4 }}>
-                {widget.type === 'News' && <NewsWidget widget={widget} />}
-
-                {isCollection(widget) && (
-                  <CustomCollection
-                    _id={widget._id}
-                    title={widget.title}
-                    type={widget.type}
-                    bookmarks={widget.bookmarks || []}
-                    bookmarkOptions={bookmarks}
-                    handleRemoveCollection={() => {
-                      handleRemoveCollection({
-                        variables: {
-                          _id: widget._id,
-                        },
-                        refetchQueries: [`getMySpace`],
-                      })
-                    }}
-                    handleEditCollection={(
-                      title: string,
-                      bookmarks?: Bookmark[]
-                    ) => {
-                      handleEditCollection({
-                        variables: {
-                          _id: widget._id,
-                          title,
-                          bookmarks,
-                        },
-                        optimisticResponse: {
-                          editCollection: {
+            data.mySpace.map((widget: Widget) => {
+              if (
+                widget.type === 'GuardianIdeal' &&
+                flags.guardianIdealCarousel
+              ) {
+                return (
+                  <Grid
+                    key={`widget_${widget._id}`}
+                    className={styles.guardianIdeal}>
+                    <GuardianIdealCarousel
+                      ideals={GuardianIdealPillars}
+                      widget={widget}
+                    />
+                  </Grid>
+                )
+              }
+
+              if (widget.type === 'News') {
```

Comment:
Question(non-blocking): Is it worth while to update this if and the one above for the guardian idea to have a function like `isCollection`? Seems like it might make it easier to test if we need to.